zha/application/platforms/sensor/__init__.py

```diff
@@ -694,6 +694,28 @@ class PolledElectricalMeasurement(ElectricalMeasurement):
     _use_custom_polling: bool = True
 
 
+@MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_ELECTRICAL_MEASUREMENT)
+class ElectricalMeasurementRMSActivePowerPhB(PolledElectricalMeasurement):
+    """RMS active power phase B measurement."""
+
+    _attribute_name = "active_power_ph_b"
+    _unique_id_suffix = "active_power_ph_b"
+    _attr_translation_key: str = "active_power_ph_b"
+    _use_custom_polling = False  # Poll indirectly by ElectricalMeasurementSensor
+    _skip_creation_if_no_attr_cache = True
+    _attr_max_attribute_name = "active_power_max_ph_b"
+
+
+@MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_ELECTRICAL_MEASUREMENT)
+class ElectricalMeasurementRMSActivePowerPhC(ElectricalMeasurementRMSActivePowerPhB):
```

Comment:
Let's remove the inheritance from `ElectricalMeasurementRMSActivePowerPhB` and change it to `PolledElectricalMeasurement`, similar to the other PR.
`_skip_creation_if_no_attr_cache = True` and `_use_custom_polling = False` have to be added then.